pkg/rancher-desktop/pages/Containers.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div class="containers">
+    <SortableTable
+      ref="containersTable"
+      :headers="headers"
+      key-field="Id"
+      :rows="rows"
+      :row-actions="false"
+      :table-actions="false"
+      :paging="true"
+      :loading="!containersList"
+    >
+      <template #col:ports="{row}">
+        <div class="port-container">
+          <a
+            v-for="port in getUniquePorts(row.Ports)"
+            :key="port"
+            target="_blank"
+            class="link"
+            @click="openUrl(port)"
+          >
+            {{ port }}
+          </a>
+        </div>
+      </template>
+    </SortableTable>
+  </div>
+</template>
+
+<script>
+import SortableTable from '@pkg/components/SortableTable';
+import { shell } from 'electron';
+import { cloneDeep } from 'lodash';
+
+import { defaultSettings } from '@pkg/config/settings';
+let hasContainers = false;
+
+export default {
+  name:       'Containers',
+  title:      'Containers',
+  components: { SortableTable },
+  data() {
+    return {
+      settings:       defaultSettings,
+      containersList: null,
+      mountTable:     false,
+      headers:
+      [
+        // INFO: Disable for now since we can only get the running containers.
+        // {
+        //   name:  'containerState',
+        //   label: this.t('containers.manage.table.header.state'),
+        //   sort:  ['containerName', 'image', 'imageName'],
+        // },
+        {
+          name:  'containerName',
+          label: this.t('containers.manage.table.header.containerName'),
+          sort:  ['containerName', 'image', 'imageName'],
+        },
+        {
+          name:  'imageName',
+          label: this.t('containers.manage.table.header.image'),
+          sort:  ['imageName', 'containerName', 'imageName'],
+        },
+        {
+          name:  'ports',
+          label: this.t('containers.manage.table.header.ports'),
+          sort:  ['ports', 'containerName', 'imageName'],
+        },
+        {
+          name:  'started',
+          label: this.t('containers.manage.table.header.started'),
+          sort:  ['si', 'containerName', 'imageName'],
+          width: 120,
+        },
+      ],
+    };
+  },
+  computed: {
+    rows() {
+      if (!this.containersList) {
+        return [];
+      }
+
+      const containers = cloneDeep(this.containersList);
```

Comment:
`cloneDeep` is entirely valid, but I've been trying to encourage using `structuredClone`[^1] as of late. Perhaps, we can consider if it will fit with our use case here.  

[^1]: https://developer.mozilla.org/en-US/docs/Web/API/structuredClone